.gitpod.yml

```diff
@@ -0,0 +1,6 @@
+ports:
+- port: 3000
+  onOpen: open-preview
+tasks:
+- init: npm install
+  command: AWS_EXECUTION_ENV=true npm start
```

Comment:
I think `AWS_EXECUTION_ENV=true` is not necessary, as the server is a normal express app.

Reply:
I needed to add `AWS_EXECUTION_ENV=true` to force `@serverless-chrome/lambda` to use its own headless Chrome binary. Without it, I think it tries to use a local Chrome, and even if you `apt-get install chromium-browser`, it will fail with strange sandbox errors.